operations/restrictions.md

```diff
@@ -343,6 +343,8 @@ Adds new restrictions with the specified conditions.
 
 Removes restrictions from the service.
 
+> **Important:** Since the [set restrictions](#set-restrictions) operation is the preffered way to add restrictions, this endpoint becomes unusable when used in conjunction with the set. This is because the set operation does not return the ids of individual restrictions. To delete restrictions added by the set restrictions operation, use the [clear restrictions](#clear-restrictions) operation.
```

Comment:
I do not think this paragraph is needed. The endpoint is _not_ unusable, because 'Get all restrictions' returns the IDs of restrictions, including restrictions created by 'Set restrictions'. I think the user can see that there are two similar operations, 'Delete restrictions' and 'Clear restrictions', and can choose which one is appropriate for their use case.